lerobot2mcap/dataset_info.py

```
@@ -0,0 +1,237 @@
+"""Dataset information parser for LeRobot datasets.
+Parses the info.json file to extract metadata about the dataset,"""
+
+import json
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+# Default values for dataset metadata
+DEFAULT_FPS = 30
+DEFAULT_CODEC = "h264"
+DEFAULT_FILE_INDEX = 0  # First file in a chunk
+DEFAULT_DATA_PATH_TEMPLATE = (
+    "data/chunk-{chunk_index:03d}/file-{file_index:03d}.parquet"
+)
+DEFAULT_VIDEO_PATH_TEMPLATE = (
+    "videos/{video_key}/chunk-{chunk_index:03d}/file-{file_index:03d}.mp4"
+)
+
+
+class DatasetInfo:
```

* Silent defaults for required fields. The code uses `.get("key", default)`. Required fields must raise an error.
* The DEFAULT_* constants (lines 11-19) should only exist if they're part of the official LeRobot dataset specification.
* Missing parquet DB version get helper function. Do we check it in this converter?

---

- addressed silent defaults and ensured only official lerobot defaults are defined as constants
- db version helper function get_codebase_version() implemented